client_end_user/shared/src/commonMain/kotlin/presentation/home/HomeScreen.kt

```diff
@@ -0,0 +1,65 @@
+package presentation.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import cafe.adriel.voyager.navigator.Navigator
+import com.beepbeep.designSystem.ui.theme.Theme
+import presentation.base.BaseScreen
+import presentation.composable.SectionHeader
+import presentation.home.composable.CuisineCard
+import resources.Resources
+
+class HomeScreen :
+    BaseScreen<HomeScreenModel, HomeScreenUiState, HomeScreenUiEffect, HomeScreenInteractionListener>() {
+
+    @Composable
+    override fun Content() {
+        initScreen(getScreenModel())
+    }
+
+    override fun onEffect(effect: HomeScreenUiEffect, navigator: Navigator) {
+        when (effect) {
+            is HomeScreenUiEffect.CuisineUiEffect -> println("Cuisine id ${effect.cuisineId}")
```

Comment:
Could we rename effect to 
NavigateToCuisineDetails?
And No need to repeat Suffix UiEffect